src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The current logic applies the long-term discount first, which is correct, but the short-term discount is applied next without checking if the long-term discount was already applied. This means that for rentals longer than 7 days, both discounts could be applied, which is likely not the intended behavior. Consider restructuring the conditions to ensure only one discount is applied based on the rental duration.

Reply:
Voce precisa aceitar esta tarefa pois ela foi guiada por vídeo do mentor da mate-academy e esta exatamente igual ele pediu para fazer